src/components/CookieProvider.ts

```diff
@@ -0,0 +1,27 @@
+import { Ajax } from 'src/libs/ajax';
+import { AbortOption } from 'src/libs/ajax/data-provider-common';
+import { azureCookieReadyStore, cookieReadyStore } from 'src/libs/state';
+
+export interface CookieProvider {
+  invalidateCookies: (options?: AbortOption) => Promise<void>;
+}
+export const cookieProvider: CookieProvider = {
+  invalidateCookies: async (options: AbortOption = {}) => {
+    const { signal } = options;
+    const cookies: string[] = document.cookie.split(';');
+    // TODO: call azure invalidate cookie once endpoint exists, https://broadworkbench.atlassian.net/browse/IA-3498
```

Comment:
Good reminder to do this for Azure. We could probably add an `invalidateCookie` endpoint in the Listener alongside `setCookie`, around [here](https://github.com/DataBiosphere/terra-azure-relay-listeners/blob/39641f8abfb5d71dc7eac8043365fc056d93c00f/service/src/main/java/org/broadinstitute/listener/relay/http/RelayedHttpRequestProcessor.java#L160)

Reply:
Is this done for azure now? Or is this something we are doing in that other ticket?